### PR TITLE
Fixes documentation

### PR DESCRIPTION
**Pull request by oanaoana**
This fixes documentation issues. 

## Reason
Wrong refs in Pseudotimestep.md and typsetting for readability in NonlinearSystem.md

## Design
Fix MOOSE markdown

## Impact
User appeal.

### CODE REVIEW

File: framework/doc/content/source/postprocessors/PseudoTimestep.md [hunk 1]
```diff
@@ -22,36 +22,36 @@ If nothing is known about the problem we recommend `initial_dt = 1` and `alpha =
...
 
 \begin{equation}
 \Delta t_k =\Delta t_{k-1} \cdot \alpha^{\frac{R_{k-1}-R_k}{R_{k-1}}}
 \end{equation}
 
 This implementation is the `RDM` method variant as found in [!citep](ceze2013pseudo), other variants are available in e.g. [!citep](bucker2009cfl).
 
-- Exponential progression (EXP), as exemplified in [fv_burgers_EXP.i](test/tests/postprocessors/pseudotimestep/fv_burgers_EXP.i),
-
+- Exponential progression (EXP)
```
Comment (cticenhour) [inline framework/doc/content/source/postprocessors/PseudoTimestep.md:41]:
```suggestion
- +Exponential progression (EXP)+
```

File: framework/doc/content/source/postprocessors/PseudoTimestep.md [hunk 2]
```diff
@@ -22,36 +22,36 @@ If nothing is known about the problem we recommend `initial_dt = 1` and `alpha =
...
 
 \begin{equation}
 \Delta t_k =\Delta t_{0} \cdot \alpha^k
 \end{equation}
 
 where $\alpha$ is a user chosen parameter, $k$ is the current iteration step.
```
Comment (cticenhour) [inline framework/doc/content/source/postprocessors/PseudoTimestep.md:47]:
```suggestion
  \begin{equation}
  \Delta t_k =\Delta t_{0} \cdot \alpha^k
  \end{equation}

  where $\alpha$ is a user chosen parameter, $k$ is the current iteration step.
```

File: framework/doc/content/source/postprocessors/PseudoTimestep.md [hunk 3]
```diff
@@ -22,36 +22,36 @@ If nothing is known about the problem we recommend `initial_dt = 1` and `alpha =
...
 
-As noted also in [!citep](bucker2009cfl) the EXP method has an infinite growth, so for this method a [!param](/Postprocessors/PseudoTimestep/max_dt) parameter may be recommended. If no [!param](/Postprocessors/PseudoTimestep/max_dt) is provided by the user then infinite growth of the timestep is not bounded and the user will be informed by a message at the console. Ideally this method is used in conjunction with a steady state detection, i.e. setting [!param](/Executioner/Transient/steady_state_detection) to `true` and a [!param](/Executioner/Transient/steady_state_tolerance).
-
+As an example this postprocessor may be implemented as
```
Comment (cticenhour) [inline framework/doc/content/source/postprocessors/PseudoTimestep.md:51]:
```suggestion
## Example Input File Syntax
```

This aligns with other object documentation sections.

File: framework/doc/content/source/systems/NonlinearSystem.md [hunk 4]
```diff
@@ -16,31 +16,43 @@ and `NonlinearEigenSystem` in [NonlinearSystemBase.md].
 ## Solving Non-linear Systems id=newtons_method
 
 Application of the finite element method converts PDE(s) into a system of
-nonlinear equations, $R_i(u_h)=0, \qquad i=1,\ldots, N$
+nonlinear equations, $R_i(u_h)=0, \quad i=1,\ldots, N$
   to solve for the coefficients $u_j, j=1,\dots,N$.
```
Comment (cticenhour) [inline framework/doc/content/source/systems/NonlinearSystem.md:20]:
```suggestion
  to solve for the coefficients $u_j, \quad j=1,\dots,N$.
```

Let's add a bit more space here too, while we're at it.

File: framework/doc/content/source/systems/NonlinearSystem.md [hunk 5]
```diff
@@ -234,37 +246,44 @@ Note that only one member is needed to represent shape functions for standard
 ### Newton for a Simple Equation id=simple_newton
 
 - Consider the convection-diffusion equation with nonlinear $k$, $\vec{\beta}$, and $f$:
-    $\begin{aligned}- \nabla\cdot k\nabla u + \vec{\beta} \cdot \nabla u = f\end{aligned}$
+
+\begin{equation}- \nabla\cdot k\nabla u + \vec{\beta} \cdot \nabla u = f
+\end{equation}
```
Comment (cticenhour) [inline framework/doc/content/source/systems/NonlinearSystem.md:251]:
```suggestion
\begin{equation}
- \nabla\cdot k\nabla u + \vec{\beta} \cdot \nabla u = f
\end{equation}
```

File: framework/doc/content/source/systems/NonlinearSystem.md [hunk 6]
```diff
@@ -234,37 +246,44 @@ Note that only one member is needed to represent shape functions for standard
...
+\begin{equation}
+\dfrac{\partial f}{\partial u_j} = \dfrac{\partial f}{\partial u_h} \dfrac{\partial u_h}{\partial u_j}=\dfrac{\partial f}{\partial u_h} \phi_j
+\end{equation}
 
 - If a functional form of $f$ is known, e.g. $f(u) = \sin(u)$, this
   formula implies that its Jacobian contribution is given by
 
-   $\frac{\partial f}{\partial u_j} = \cos(u_h) \phi_j$
+\begin{equation} \dfrac{\partial f}{\partial u_j} = \cos(u_h) \phi_j
+\end{equation}
```
Comment (cticenhour) [inline framework/doc/content/source/systems/NonlinearSystem.md:286]:
```suggestion
\begin{equation}
\dfrac{\partial f}{\partial u_j} = \cos(u_h) \phi_j
\end{equation}
```

File: framework/doc/content/source/systems/NonlinearSystem.md [hunk 7]
```diff
@@ -296,7 +319,9 @@ $\begin{aligned} J_{ij}(u_h) &= \left(\nabla\psi_i, \frac{\partial k}{\partial u
 
 
 - This action can be approximated by:
-    $\mathbf{J}\vec{v} \approx \frac{\vec{R}(\vec{u} + \epsilon\vec{v}) - \vec{R}(\vec{u})}{\epsilon}$
+
+\begin{equation}\mathbf{J}\vec{v} \approx \dfrac{\vec{R}(\vec{u} + \epsilon\vec{v}) - \vec{R}(\vec{u})}{\epsilon}
+\end{equation}
```
Comment (cticenhour) [inline framework/doc/content/source/systems/NonlinearSystem.md:324]:
```suggestion
\begin{equation}
\mathbf{J}\vec{v} \approx \dfrac{\vec{R}(\vec{u} + \epsilon\vec{v}) - \vec{R}(\vec{u})}{\epsilon}
\end{equation}
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
A few suggestions. Thanks for adjusting the previous formatting!

**Review by GiudGiud — APPROVED** [review]
looks a lot better!

**Comment by loganharbour** [comment]
Please use batching for review changes (see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request for more info) to avoid extraneous commits and extraneous civet events in the future

**Review by cticenhour — APPROVED** [review]
Thanks @oanaoana!

**Comment by oanaoana** [comment]
> Please use batching for review changes (see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request for more info) to avoid extraneous commits and extraneous civet events in the future

Aye Aye Sir!!!

**Pull request merged**